MAUI/TreeView/working-with-treeview.md

```diff
@@ -102,6 +102,32 @@ private void TreeView_ItemLongPress(object sender, ItemLongPressEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### KeyDown event
+
+The `KeyDown` event is triggered when a key is pressed while the [SfTreeView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TreeView.html) is in focus. The `KeyPressEventArgs` has the following members that provide information for the `KeyDown` event:
+
+ * `Key`: Returns the currently pressed key.
+ * `IsShiftKeyPressed`: Indicates whether the Shift key is in pressed state.
+ * `IsCtrlKeyPressed`: Indicates whether the Control key is in pressed state.
+ * `IsAltKeyPressed`: Indicates whether the Alt key is in pressed state.
+ * `IsCommandKeyPressed`: Indicates whether the Command key is in pressed state.
+ * `Handled`: Gets or sets whether the event is handled or not.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfTreeView x:Name="treeView" KeyDown="treeView_KeyDown"/>
+{% endhighlight %}
+{% highlight c# %}
+
+treeView.KeyDown += this.TreeView_KeyDown;

```

Comment:
no need to show event wiring line in C#.
and rename method name as OnTreeViewKeyDown in both xaml and C#

Reply:
done